SPECS/perl/perl.spec

```diff
@@ -74,6 +78,10 @@ make test TEST_SKIP_VERSION_CHECK=1
 %{_mandir}/*/*
 
 %changelog
+
```

Comment:
Can you make the changes suggested by the linter? I looked at the linter output and all of the changes look reasonable.